### PR TITLE
Drop Rails 4.2 support

### PR DESCRIPTION
**Pull request by unixcharles**
As the title say, see issue [#149](https://github.com/activemerchant/payment_icons/issues/149).

### CODE REVIEW

**Comment by NathanPJF** [comment]
The `Gemfile.lock` file generated has the following:

```
PATH
  remote: .
  specs:
    payment_icons (1.0.4)
      frozen_record
      rails (>= 4.1, < 6.0)
      sass
```

Rails `>=4.1` is listed as supported.  

To change this, the `gemspec` file needs to be updated: https://github.com/activemerchant/payment_icons/blob/master/payment_icons.gemspec#L19

File: payment_icons.gemspec [hunk 1]
```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
 
   s.add_dependency 'frozen_record'
-  s.add_dependency 'rails', '>= 4.1', '< 6.0'
+  s.add_dependency 'rails', '>= 5.0', '< 6.0'
```
Comment (NathanPJF) [inline payment_icons.gemspec]: Pushed the update myself

Comment (unixcharles) [reply payment_icons.gemspec]: Should we enforce it? Even if its not supported it should still work.

Comment (unixcharles) [reply payment_icons.gemspec]: nvm, let just be explicit

**Pull request merged**